### ISSUE TITLE
Feature request: method-update-pushArray

### ISSUE THREAD

**Issue by wujekbogdan** [Meteor-Community-Packages/meteor-autoform#932]
It would be great if there was a possibility to use  update-pushArray together with meteormethod.

**Comment by pocd** [comment]
I was just stumbling across the same thing. I needed to do my own server side pre-pushArray inserts but there is no functionality for that.

**Comment by aldeed** [comment]
I'm not sure why this would be necessary since you can just take the `doc` that is sent to your method for a "method" form and `$push` it in your method.

**Comment by cryve** [comment]
Currently dealing with the same issue while refactoring my autoforms to call validated-methods. The "method" approach works fine, but I only got it running by using a Non-collection form based on a sub-schema and a "before" hook for inserting the docId into the "doc" that is send to the method.

Maybe I did things too complicated, but I'd love a solution like the "method-update" with the "singleMethodArgument=true" option, which plays very well with validated-methods and produce lean code. 

So a "method-update-pushArray" delivering a modifier using "$push" would be great!
